### ISSUE TITLE
Code cannot be read and edited using a screen reader in Android web version of VS code.

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#34673]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: web
- OS Version: android 10

Steps to Reproduce:

1. Turn on TalkBack from settings >accessibility
2. open any file in web version of VS code.
3. Try to read the code inside the file with screen reader. Whenever the edit box is focused, the screen reader only announces name of the file with the edit box and does not announces any code written in the file. Hence the code cannot be edited as well. Also after typing some code if enter is pressed (in the mobile keyboard)the edit box becomes blank for the screen reader and it is not possible to read previous lines of that file.
